test/Factory/Config/YAMLConfigProviderTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Tests\Factory\Config;
+
+use Migrator\Factory\Config\YAMLConfigProvider;
+use PHPUnit_Framework_TestCase;
+use RuntimeException;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class YAMLConfigProviderTest
+ * @package Tests\Factory\Config
+ */
+class YAMLConfigProviderTest extends PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        if (!class_exists(Yaml::class)) {
```

Comment:
I believe "symfony/yaml" should also be in the "require-dev" list in composer.json. That would make this check unnecessary.